components/crud-web-apps/jupyter/backend/apps/common/utils.py

```diff
@@ -132,4 +137,5 @@ def notebook_dict_from_k8s_obj(notebook):
         "memory": cntr["resources"]["requests"]["memory"],
         "volumes": [v["name"] for v in cntr["volumeMounts"]],
         "status": status.process_status(notebook),
+        "dashLink": dashLink
```

Comment:
is there a place where this term "dashLink" came from? I'd expect it to be called debugURL or something like that.